src/BinaryParsers/ElfBinary/Dwarf/DwarfCompilationUnit.cs

```diff
@@ -194,6 +195,12 @@ private void ReadData(IDwarfBinary dwarfBinary, DwarfMemoryReader debugData, Dwa
                             attributeValue.Value = debugStrings.ReadString(offsetStrp);
                             break;
 
+                        case DwarfFormat.StrpSup:
```

Comment:
>StrpSup [](http://example.com/codeflow?start=41&length=7)

This case wasn't handled previously. It wasn't causing our crash, as it happens, but I fixed it anyway.